prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_SR_1_20230220-0001.hl7

```diff
@@ -0,0 +1 @@
OBR|5||21:AA:B0029251S.5^M12776123.5|VGP-75^VITEK2 AST-GP75^L|||20280807163601-0600|20280807163601-0600||ER|||SEPTIC SHOCK|||1871759365^Willison^Christopher^Beau^MD^^^^NPI&2.16.840.1.114222.4.1.144&ISO^L^^^NPI^RML|^WPN^PH^^1^303^4362727^^Hospital Line^^^+1 303 436 2727|21:AA:B0029251Sm12776123|09339017||ROSE MEDICAL CENTER (MCOE)|20210810062500-0600|||P|600-7&&LN&BCEXT&EXTENDED BLOOD CULTURE&L^1^ENTRAFVRE|||^21:AA:B0029251Sm12776123&RML||R6521^SEVERE SEPSIS WITH SEPTIC SHOCK^I10^R6521^SEVERE SEPSIS WITH SEPTIC SHOCK^I10
OBX|1|SN|28-1^Ampicillin [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^AM^AMPICILLIN^L|1|=^16|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L||R|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^R|1033122247^Chahal^Shalini^^MD^Dr.^^^RML^L^^^FI^ROSE MEDICAL CENTER 
OBX|2|ST|7018-5^Gentamicin.high potency [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^GMS^GENTAMICIN SYNERGY SCREEN^L|1|SYN-S|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L||S|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^S|1033122247^Chahal^Shalini^^MD^Dr.^^^RML^L^^^FI^ROSE MEDICAL CENTER 
OBX|3|SN|29254-0^Linezolid [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^LZD^LINEZOLID^L|1|=^2|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L||S|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^S|1033122247^Chahal^Shalini^^MD^Dr.^^^RML^L^^^FI^ROSE MEDICAL CENTER 
OBX|4|ST|6933-6^Streptomycin.high potency [Susceptibility] by Minimum inhibitory concentration (MIC)^LN^STS^STREPTOMYCIN SYNERGY SCREEN^L|1|SYN-S|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L||S|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^S|1033122247^Chahal^Shalini^^MD^Dr.^^^RML^L^^^FI^ROSE MEDICAL CENTER 
OBX|5|SN|19000-9^Vancomycin [Susceptibility]^LN^VA^VANCOMYCIN^L|1|>=^32|ug/mL^MicroGrams Per MilliLiter [Mass Concentration Units]^UCUM^ug/mL^ug/mL^L||R|||P|||20280807163601-0600|MCOE^ROSE MEDICAL CENTER (MCOE)^LB||||20280807163601-0600||||ROSE MEDICAL CENTER (MCOE)^L|4567 E. NINTH AVENUE^^DENVER^IG^80220^USA^B^^R|1033122247^Chahal^Shalini^^MD^Dr.^^^RML^L^^^FI^ROSE MEDICAL CENTER 
SPM|1|09339017^M12776123||119303007^Microbial isolate^SCT^ISLT^ISOLATE^L^07/31/2012^5.67||||RA^RIGHT ARM^L^5.67||||||RA|||20280808092805-0600^20280808092805-0600|20210804001800-0600||||||||3.0|^BACTEC BLOOD CULTURE BOTTLES

```

Comment:
You added these two files, but you did not add them to the test config, so they are tested.  See the config file.

Reply:
Updated TranslationTests to support FHIR to HL7 and added the test to the config file.